python/tvm/relay/transform/transform.py

```diff
@@ -1204,7 +1204,7 @@ def FakeQuantizationToInteger():
         x    w
         |    |
         dq   dq
-         \   /
+         \\   /
```

Comment:
Nit: Why is your code formatter screwing with my ASCII Art? :grin:

Reply:
Haha yeah it is intentional. Otherwise the unhappy pytest reports a warning demanding escape characters

Reply:
Hmm, I wonder why I didn't see that in CI when I added this.

Reply:
Same here when I run the pytest, got a warning for this.